canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -107,12 +108,20 @@ class PropertiesMain extends React.Component {
 				(newProps.propertiesInfo.formData && !isEqual(newProps.propertiesInfo.formData, this.props.propertiesInfo.formData)) ||
 				(newProps.propertiesInfo.parameterDef && !isEqual(newProps.propertiesInfo.parameterDef, this.props.propertiesInfo.parameterDef)) ||
 				(newProps.propertiesInfo.appData && !isEqual(newProps.propertiesInfo.appData, this.props.propertiesInfo.appData))) {
+				// Save prevId first before setting the new one from propertiesInfo
+				this.propertiesController.prevId = this.propertiesController.getId();
```

Comment:
Why do we need to store the id?  Can we just compare the new vs prev properties like we do other the parameterDef.

Reply:
We store everything under `propertiesInfo` in `propertiesController` - https://github.com/elyra-ai/canvas/blob/main/canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx#L59C3-L60C84

Followed the same pattern for `propertiesInfo.id`.

Reply:
Right, but we really don't need this for later on like the others.